src/arbiter.rs

```diff
@@ -97,7 +128,10 @@ impl Arbiter {
 
         // start arbiter
         let ctx = Context::with_receiver(rx);
-        let fut = ctx.into_future(Arbiter { stop: None });
+        let fut = ctx.into_future(Arbiter {
+            stop: None,
+            stop_system_on_panic: true, // If the system Arbiter panics, stop the system.
```

Comment:
I figured that if this Arbiter failed, it'd be worth stopping. But it'd probably stop anyways with a panic.